dbinit.sql

```diff
@@ -4,130 +4,97 @@ SET sql_safe_updates = false;
-    CONSTRAINT fk_city_ref_users FOREIGN KEY (rider_city, rider_id) REFERENCES users(city, id),
-    CONSTRAINT fk_vehicle_city_ref_vehicles FOREIGN KEY (city, vehicle_id) REFERENCES vehicles(city, id),
-    INDEX rides_auto_index_fk_city_ref_users (rider_city ASC, rider_id ASC) PARTITION BY LIST (rider_city) (
-        PARTITION us_west VALUES IN (('seattle'), ('san francisco'), ('los angeles')),
-        PARTITION us_east VALUES IN (('new york'), ('boston'), ('washington dc')),
-        PARTITION europe_west VALUES IN (('amsterdam'), ('paris'), ('rome'))
-    ),
-    INDEX rides_auto_index_fk_vehicle_city_ref_vehicles (city ASC, vehicle_id ASC) PARTITION BY LIST (city) (
-        PARTITION us_west VALUES IN (('seattle'), ('san francisco'), ('los angeles')),
-        PARTITION us_east VALUES IN (('new york'), ('boston'), ('washington dc')),
-        PARTITION europe_west VALUES IN (('amsterdam'), ('paris'), ('rome'))
-    ),
-    FAMILY "primary" (id, city, rider_id, rider_city, vehicle_id, start_location, end_location, start_time, end_time, length)
-)  PARTITION BY LIST (city) (
-        PARTITION us_west VALUES IN (('seattle'), ('san francisco'), ('los angeles')),
-        PARTITION us_east VALUES IN (('new york'), ('boston'), ('washington dc')),
-        PARTITION europe_west VALUES IN (('amsterdam'), ('paris'), ('rome'))
-);
-ALTER PARTITION europe_west OF INDEX movr.public.rides@primary CONFIGURE ZONE USING
-    constraints = '[+region=gcp-europe-west1]';
-ALTER PARTITION us_east OF INDEX movr.public.rides@primary CONFIGURE ZONE USING
-    constraints = '[+region=gcp-us-east1]';
-ALTER PARTITION us_west OF INDEX movr.public.rides@primary CONFIGURE ZONE USING
-    constraints = '[+region=gcp-us-west1]';
-ALTER PARTITION europe_west OF INDEX movr.public.rides@rides_auto_index_fk_city_ref_users CONFIGURE ZONE USING
-    constraints = '[+region=gcp-europe-west1]';
-ALTER PARTITION us_east OF INDEX movr.public.rides@rides_auto_index_fk_city_ref_users CONFIGURE ZONE USING
-    constraints = '[+region=gcp-us-east1]';
-ALTER PARTITION us_west OF INDEX movr.public.rides@rides_auto_index_fk_city_ref_users CONFIGURE ZONE USING
-    constraints = '[+region=gcp-us-west1]';
-ALTER PARTITION europe_west OF INDEX movr.public.rides@rides_auto_index_fk_vehicle_city_ref_vehicles CONFIGURE ZONE USING
-    constraints = '[+region=gcp-europe-west1]';
-ALTER PARTITION us_east OF INDEX movr.public.rides@rides_auto_index_fk_vehicle_city_ref_vehicles CONFIGURE ZONE USING
-    constraints = '[+region=gcp-us-east1]';
-ALTER PARTITION us_west OF INDEX movr.public.rides@rides_auto_index_fk_vehicle_city_ref_vehicles CONFIGURE ZONE USING
-    constraints = '[+region=gcp-us-west1]'
-;
+    CONSTRAINT fk_city_ref_users FOREIGN KEY (rider_id) REFERENCES users(id),
+    CONSTRAINT fk_vehicle_ref_vehicles FOREIGN KEY (vehicle_id) REFERENCES vehicles(id)
+    );
+
+
+ALTER TABLE rides ADD COLUMN region crdb_internal_region AS (
```

Comment:
do we ever make this table `REGIONAL BY ROW`? seems like it is REGIONAL BY TABLE at the moment.
we should do this inline, i.e.

```
CREATE TABLE users (

   ...,
   region crdb_internal_region  CASE WHEN city = 'amsterdam' THEN 'gcp-europe-west1'
       WHEN city = 'paris' THEN 'gcp-europe-west1'
       WHEN city = 'rome' THEN 'gcp-europe-west1'
       WHEN city = 'new york' THEN 'gcp-us-east1'
       WHEN city = 'boston' THEN 'gcp-us-east1'
       WHEN city = 'washington dc' THEN 'gcp-us-east1'
       WHEN city = 'san francisco' THEN 'gcp-us-west1'
       WHEN city = 'seattle' THEN 'gcp-us-west1'
       WHEN city = 'los angeles' THEN 'gcp-us-west1'
       ELSE 'gcp-us-east1',
     
       ....
) LOCALITY REGIONAL BY ROW; -- optionally REGIONAL BY ROW AS "region"

same with the others

Reply:
My mistake. Fixed for `rides`.

Also, moved the `region` column definition to the `CREATE TABLE` statement for all tables.